lib/Cake/Network/CakeResponse.php

```diff
@@ -1336,7 +1336,7 @@ public function file($path, $options = array()) {
 			'download' => null
 		);
 
-		if (strpos($path, '..') !== false) {
+		if (strpos($path, '../') !== false || strpos($path, '..\\') !== false) {
```

Comment:
But hardcoded / should never be used on WIN. It will eventually fail somewhere, especially when you compare paths (as I found out the hard way with Themes logic).